ofbizDemo/widget/OfbizDemoScreens.xml

```diff
@@ -54,20 +53,166 @@
+                <set field="titleProperty" value="OfbizDemoAddOfbizDemoFtl" />
+                <set field="headerItem" value="addOfbizDemoFtl" />
+                <script location="component://ofbizDemo/webapp/ofbizDemo/WEB-INF/actions/crud/ListOfbizDemo.groovy" />
             </actions>
             <widgets>
                 <decorator-screen name="OfbizDemoCommonDecorator" location="${parameters.mainDecoratorLocation}">
                     <decorator-section name="body">
-                        <label style="h4" text="${uiLabelMap.OfbizDemoListOfbizDemos}"/>
+                        <label style="h4" text="${uiLabelMap.OfbizDemoListOfbizDemos}" />
+                        <platform-specific>
+                            <html>
+                                <html-template location="component://ofbizDemo/webapp/ofbizDemo/crud/ListOfbizDemo.ftl" />
+                            </html>
+                        </platform-specific>
+                        <label style="h4" text="${uiLabelMap.OfbizDemoAddOfbizDemoFtl}" />
                         <platform-specific>
-                            <html><html-template location="component://ofbizDemo/webapp/ofbizDemo/crud/ListOfbizDemo.ftl"/></html>
+                            <html>
+                                <html-template location="component://ofbizDemo/webapp/ofbizDemo/crud/AddOfbizDemo.ftl" />
+                            </html>
                         </platform-specific>
-                        <label style="h4" text="${uiLabelMap.OfbizDemoAddOfbizDemoFtl}"/>
+                    </decorator-section>
+                </decorator-screen>
+            </widgets>
+        </section>
+    </screen>
+    <!--    CreateProductionRun Screen-->
+    <screen name="CreateProduction">
+        <section>
+            <actions>
+                <set field="headerItem" value="CreateProduction" />
+                <set field="titleProperty" value="CreateProduction" />
+            </actions>
+            <widgets>
+                <decorator-screen name="main-decorator" location="${parameters.mainDecoratorLocation}">
+                    <decorator-section name="body">
+                        <screenlet title="${uiLabelMap.CreateProduction}">
+                            <include-form name="CreateProduction" location="component://ofbizDemo/widget/OfbizDemoForms.xml" />
+                        </screenlet>
+                    </decorator-section>
+                </decorator-screen>
+            </widgets>
+        </section>
+    </screen>
+    <!--        FindProductionRun-->
+    <screen name="FindProductionRun">
+        <section>
+            <actions>
+                <set field="headerItem" value="FindProductionRun" />
+                <set field="titleProperty" value="findProductionRun" />
+                <entity-and entity-name="Facility" list="facilities" >
+                        <field-map field-name="facilityId" value=""/>
```

Comment:
this field-map is not required
you can close the entity-and <entity-and entity-name="Facility" list="facilities" />